File: src/AntShares.SmartContract.Template/Contract1.cs
using AntShares.SmartContract.Framework;
using AntShares.SmartContract.Framework.Services.AntShares;
using System;
using System.Numerics;

namespace $safeprojectname$
{
	public class Contract1 : FunctionCode
{
    public static void Main()
    {
        Storage.Put(StorageContext.Current, "Hello", "World");
    }
}
}

File: templates/Template.VB/Contract1.vb
Imports Neo.SmartContract.Framework
Imports Neo.SmartContract.Framework.Services
Imports System
Imports System.Numerics

Public Class $itemname$ : Inherits SmartContract
    Public Shared Function Main() As Boolean
        Storage.Put(Storage.CurrentContext, "Hello", "World")
        Return True
    End Function
End Class
